inference-engine/ie_bridges/c/samples/hello_classification/main.c

```diff
@@ -1,18 +1,28 @@
-// Copyright (C) 2018-2020 Intel Corporation
+// Copyright (C) 2018-2021 Intel Corporation
 // SPDX-License-Identifier : Apache-2.0
 //
 
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
-#include <opencv_c_wraper.h>
+
 #include <c_api/ie_c_api.h>
+#include <opencv_c_wraper.h>
 
+/**
```

Comment:
Are we decided to use Javadoc style in addition to Google Google C++ Style?

Reply:
The style will be chosen and checked later with automatic pipeline.